### PR TITLE
feat(parental-leave): change employment flow

### PR DESCRIPTION
**Pull request by Thornvik**
## What

Adding changes to the employment flow

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts [hunk 1]
```diff
@@ -403,11 +405,64 @@ export const ParentalLeaveForm: Form = buildForm({
...
+                    },
+                    {
+                      label: unemploymentBenefits.stéttarfélagi,
+                      value: unemploymentBenefits.stéttarfélagi,
+                    },
+                    {
+                      label: unemploymentBenefits.sjúkratryggingarÍslands,
+                      value: unemploymentBenefits.sjúkratryggingarÍslands,
+                    },
+                  ],
```
Comment (qdd1) [inline libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts:459]: I think there are more than just these 3 which applicant could receive money from. Add 1 more option call 'Others'? ( Need clarification from VMST )

File: libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts [hunk 2]
```diff
@@ -403,11 +405,64 @@ export const ParentalLeaveForm: Form = buildForm({
...
+                  condition: (answers) =>
+                    (answers as {
+                      employer: {
+                        isSelfEmployed: string
+                      }
+                    })?.employer?.isSelfEmployed === NO,
+                }),
+                buildSelectField({
+                  id: 'unemploymentBenefits',
+                  title: 'Hvaðan ertu að þyggja bætur?',
```
Comment (qdd1) [inline libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts:445]: should be 'Hvaðan ertu að þiggja bætur?'

File: libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts (hunk 1 above)
Comment (Thornvik) [reply libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts:459]: These are the three we have on the jira board, guess we need to ask VMST

**Review by karenbjorg — COMMENTED** [review]
lgtm! Added some minor changes to the texts

**Review by karenbjorg — APPROVED** [review]

**Pull request closed without merging**